tests/assets/tiny_bpe_vocab.json

```diff
@@ -0,0 +1,2002 @@
+{
```

Comment:
I've not looked too closely at where this is used, but could we minify this? : )

Reply:
I follow the setting in `tiktoken_small.model` in assets (vocab_size == 2000).
I will reduce it to 500.

Reply:
Sorry @fyabc, my meaning was to use something like https://codebeautify.org/jsonminifier which just removes any unnecessary characters and formatting. The vocab size itself was probably okay!